package/samplers/hebo_sampler/README.md

```diff
@@ -0,0 +1,39 @@
+---
+author: HideakiImamura
+title: HEBO (Heteroscedastic and Evolutionary Bayesian Optimisation)
+description: HEBO addresses the problem of noisy and heterogeneous objective functions by using a heteroscedastic Gaussian process and an evolutionary algorithm.
+tags: ["sampler"]
```

Comment:
Why don't you add some tags, such as "Bayesian optimization," to provide more information about this sampler?